lib/config/presets/internal/config.ts

```diff
@@ -15,6 +15,20 @@ export const presets: Record<string, Preset> = {
       'workarounds:all',
```

Comment:
Reminder: this PR targets `main`. We probably want to create a `v36` branch, and merge this PR into the `v36` branch. 😉